libs/langgraph/langgraph/graph/state.py

```diff
@@ -415,6 +424,57 @@ def add_edge(self, start_key: Union[str, list[str]], end_key: str) -> Self:
         self.waiting_edges.add((tuple(start_key), end_key))
         return self
 
+    def add_sequence(
+        self,
+        nodes: Sequence[Union[RunnableLike, tuple[str, RunnableLike]]],
+        from_nodes: Optional[list[str]] = None,
+    ) -> Self:
+        """Add a sequence of steps that will be executed in the provided order.
```

Comment:
Maybe one more sentence about from_nodes?

```suggestion
        """Add a sequence of nodes that will be executed in the provided order.
```